src/Scene.h

```diff
@@ -7,6 +7,9 @@
 #include "Frog.h"
 #include "TextCounter.h"
 #include "Background.h"
+#include "CircleCollisionComponent.h"
+#include "EnemyEntity.h"
+#include "DuckyMath.h"
```

Comment:
can be moved to cpp

Reply:
only "CircleCollisionComponent.h" or any else?

Reply:
I think also DuckyMath, but i gave a separate comment for that 🤔